api-catalog-ui/frontend/src/components/Wizard/WizardComponents/WizardInputs.jsx

```diff
@@ -25,7 +25,79 @@ import DeleteIcon from '@material-ui/icons/Delete';
 class WizardInputs extends Component {
     constructor(props) {
         super(props);
-        this.state = {};
+        let upYaml;
```

Comment:
Could you refactor all the below code? You can create a function which contains it and call it here. Something like:
```suggestion
       const targets = this.createTargets();
```